src/pages/Introduce/introduce.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./introduce.css";
+import { Link } from "react-router-dom";
+
+const handleInput = () => {
+  console.log("input 태그 동작");
+};
+const handleButton = () => {
+  console.log("button 태그 동작");
+};
+
+const Introduce = () => {
+  return (
+    <>
+      <div id="wrap">
```

Comment:
```suggestion
      <div className="wrap">
```
id 선택자는 지양해주세요!
id 선택자는 페이지에서 한번만 사용이 되어야 하는데 CRA로 설치한 프로젝트는 SPA기 때문에
페이지가 단 하나로 이루어진 어플리케이션입니다!
이럴 경우 id 선택자보다는 className으로 선택자를 적용해주세요!